docs/index.md

```diff
@@ -39,17 +42,29 @@ The image below shows the timing around how the Mobi Planner turns Bookings and
 ## Amazon Web Services (AWS) and API Interfaces
 
 - **Inputs:** [Bookings and Flights](#bookings-and-flights) are streamed to Mobi via an AWS Kinesis Data Stream
+  - Jamie: The way this will work is still kind of an open question...  Under the covers I think that we will want to
+    own our own Kinesis stream for this--but I'm not sure exactly how we want to expose this to them.  One relevant
+    question is: does CyberLogic themselves use AWS?
 - **Outputs:** [Trips](#trips) are streamed back to the client via an AWS Kinesis Data Stream
+  - Jamie: Same question as inputs
 - **Data Validation:** Any data issues identified in data validation checks are reported via AWS SNS. This includes [Kinesis Ingestion Data Validation](#kinesis-ingestion-data-validation) and [Pre-planning Data Validation](#pre-planning-data-validation).
 - **APIs:** Mobi REST [APIs](#apis) can be called by the client in order to make adjustments to plans as needed. These API calls can be triggered via the client's interface for staff, e.g. via buttons in a web portal that shows the Trips. If requests cannot be parsed, then they will return a relevant HTTP response status code. If requests are parsed successfully but an issue prevents the Mobi Planner from acting on the API call, [Invalid and Infeasible Messages](#invalid-and-infeasible-messages) will be sent via AWS Simple Notification Service (SNS).
 - **Master Data:** [Master Data](#master-data) is relatively static data that includes information about physical places and the Business Rules that should apply to relevant Bookings during planning. The client can provide REST APIs for Mobi to call regularly to update Master Data, and can optionally send updates via an AWS Data Stream as well with specific information about what Bookings should be updated. These mechanisms are described further in [Sending Master Data](#sending-master-data).
+  - Jamie: This is a totally open question...  There are many ways we could do this (e.g. should we pull data from them,
+    or should they push data to us?).  Here I think we have the opportunity to see what type of mechanism will be
+    easiest for them, and to come up with a generalized mechanism around that.
```

Comment:
@jcgentry I brought this up in the eng channel - I thought the consensus was we wanted to keep it as is? At least for now